### ISSUE TITLE
USB +5V is incorrectly connected to "V+" signal

### ISSUE THREAD

**Issue by batchku** [cmuartfab/eagle#25]
USB +5V should be called "VUSB", not "V+"

![image](https://user-images.githubusercontent.com/518356/36947574-e1fe89dc-1f9b-11e8-862c-c4947ca54c88.png)

**Comment by nkakarla1** [comment]
V+ connected to USB 5V changed to VUSB. There is another V+ connected to LED2. Please confirm if that too needs to be changed to VUSB.

**Comment by Preetisar** [comment]
Yes LED 2 is the charging LED. So it needs to be connected to VUSB as well.

**Comment by nkakarla1** [comment]
V+ connected to USB 5V changed to VUSB in both the boards as necessary.
